website_no_crawler/data/data.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version='1.0' encoding='utf-8' ?>
 <odoo>
     <!-- Add add crawler configuration parameter -->
-    <record id="set_ribbon_color" model="ir.config_parameter">
+    <record id="set_crawler_disable" model="ir.config_parameter">
         <field name="key">webiste.crawler.disable</field>
```

Comment:
Shouldn't this be `website.crawler.disable` instead of `webiste.crawler.disable`?

Reply:
And why adding a system parameter?

Reply:
sorry not getting the point 😬

Reply:
got it, I'm blind ..

Reply:
> And why adding a system parameter?

Added to allow setting based on environment using server env stuff

Reply:
I hate such options that overcomplicate all modules. Just don't install the module...

Reply:
you're right, but if you have a restore from a db coming from another env, let's say from production to stage you've to remember to uninstall it each time ..

Reply:
Yes, but you automate that thing, not the other. On sysadmin side, bloating everything through environment variables makes difficult to follow a propagation problem or to trace things, and on Odoo side, to complicate code or to happen this kind of things.

Reply:
Well, the module is already merged and approve by others, so I'm not going to undo it, but please take it into account for future "new features".

Reply:
I'll surely do. Thanks